packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenuItemsContainer.tsx

```diff
@@ -38,18 +36,13 @@ const StyledDropdownMenuItemsInternalContainer = styled.div`
 
 export const DropdownMenuItemsContainer = ({
   children,
-  hasMinHeight,
   hasMaxHeight,
 }: {
   children: React.ReactNode;
-  hasMinHeight?: boolean;
   hasMaxHeight?: boolean;
 }) => {
   return (
-    <StyledDropdownMenuItemsExternalContainer
-      hasMaxHeight={hasMaxHeight}
-      hasMinHeight={hasMinHeight}
```

Comment:
@FelixMalfait Was this causing any issues? Because this was fix for #7959 With the Removal of this, I can see this issue happening again.

Reply:
@harshrajeevsingh yes sorry I realized while reviewing this PR that this wasn't the right fix for #7959. It's not very pretty to have a constant height and that was not the desired outcome. You can see in this PR how I introduced dropdown placement instead to keep the input fixed but still have it be positionned properly (to the top or toward the bottom) depending on the space available

Reply:
Note what I did was improve the design based on placement, but the placement still changes, it's less of an issue because the focus area (input) stay in the same place. But if we wanted to go further we could have forced the placement to stay still also in this situation (but without min height, through the tools provided by the useFloatting hook)

Reply:
@FelixMalfait Ok got it. Thanks for the input.  : )
Also, I found another issue. While searching, the "No result" flashes. Have a look: 
[Screencast from 2024-10-28 12-41-10.webm](https://github.com/user-attachments/assets/0ee343ef-afe6-46a9-afb2-21ba61c176bb)

Do you think this issue should be raised?